.env

```diff
@@ -1 +1 @@
-PYTHONPATH=src
+PYTHONPATH=src;integration_tests
```

Comment:
Will this work on Windows still if it's change to `PYTHONPATH=src:integration_tests` (semicolon -> colon)?

Reply:
Yes, fixed.